onadata/libs/utils/csv_import.py

```diff
@@ -628,16 +628,16 @@ def submission_xls_to_csv(xls_file):  # noqa
         # convert excel dates(floats) to datetime
         for date_column_index in date_columns:
             try:
-                row_values[date_column_index] = (
-                    row_values[date_column_index].strftime("%Y-%m-%d").isoformat()
-                )
+                row_values[date_column_index - 1] = row_values[
```

Comment:
Could we add a test case on the actual date string value from data?

Reply:
done!